starter-code/database.js

```diff
@@ -133,8 +133,13 @@ class Database {
         // productName is the name of the product that we want to buy
         // Think if you may need to implement two queries chained
         // remeber once it's finish to comment callback('Error buying product');
-        
-        callback('Error buying product');
+        database.collection('users').updateOne(
+          {'firstName':userFirstName},
+          {$set:{
```

Comment:
If you use $set, you will replace the array every time you call the function. Then, you only can have one element in shoppingCart. Instead, use $push (and pass an element, not an array), that way you add the element into the array:
```
$push: {'shoppingCart':productName}
```